runtime/queries/metricsview.go

```diff
@@ -292,6 +292,327 @@ func buildFilterClauseForCondition(mv *runtimev1.MetricsViewSpec, cond *runtimev
 	return fmt.Sprintf("AND (%s) ", condsClause), args, nil
 }
 
+type identifier struct {
+	mapped string
+	unnest bool
+}
```

Comment:
1. Since so many different things are "identified" in these queries, perhaps call it `columnIdentifier` instead of `identifier`?
2. Maybe change `mapped` to `safeName` or `sqlName` to clarify what it contains?

Reply:
It more like expression. Updated to be explicit.